examples/binaries/proxy/Cargo.toml

```diff
@@ -7,7 +7,7 @@ license = "GPL-3.0"
 workspace = "../../../"
 
 [dependencies]
-gstd = { workspace = true, features = ["debug"] }
+gstd = { workspace = true }
```

Comment:
I am going to the same for all examples right before mering to the `master`

Reply:
still need this here, maybe do it here: #2602

Reply:
@SabaunT What do you mean? Why do we need building with the `debug` feature all the time?